src/renderer/components/list/table-theme.module.scss

```diff
@@ -0,0 +1,78 @@
+.tableTheme {
```

Comment:
Hmm. Should something called "list" know of something called "table-theme"? Feels a bit off.